config/locales/sumofus.de.yml

```diff
@@ -23,6 +23,7 @@ de:
     card_declined: Der Zahlungsdienst hat Ihre Kreditkarte nicht akzeptiert. Bitte wählen Sie eine andere Zahlungsmethode.
     unknown_error: Unbekannter Fehler. Unser Team wurde benachrichtigt. Bitte überprüfen Sie Ihre Eingaben oder wählen Sie eine andere Zahlungsmethode.
     fine_print: "SumOfUs ist eine eingetragene Non-Profit-Organisation (501(c)4) mit Sitz in Washington DC, USA. Spenden an SumOfUs können nicht von der Steuer abgesetzt werden. Für mehr Informationen schreiben Sie uns bitte eine E-Mail an info@sumofus.org."
+    confirmation: Donation received # FIXME - STILL IN ENGLISH!
```

Comment:
yeah - there's no germans online and we're not fundraising in german, so this way the english fix can go out on prod for the test tomorrow